### ISSUE TITLE
WP: add option for enable Invoice sections

### ISSUE THREAD

**Issue by shendel** [swaponline/MultiCurrencyWallet#3877]
Allow users enable invoice sections

**Comment by vptteam** [comment]
This is vital

**Comment by shendel** [comment]
Update plugin and re-check

-- enable invoices
![image](https://user-images.githubusercontent.com/1880211/103141236-d260d480-4702-11eb-80f6-ff619dd4187f.png)

-- get link for invoice (this link send to other user, and user send to you invoice for pay)
click on dots near wallet (supported eth and btc - all types) - select get link for invoice
![image](https://user-images.githubusercontent.com/1880211/103141258-22d83200-4703-11eb-9ef6-0d323edb3a58.png)

Or - click on dots near wallet - select Invoice - this is manual invoice

All invoming invoices you can see in man "Transaction" section beetween coin transaction and swap history

Invoices has four state
![image](https://user-images.githubusercontent.com/1880211/103141344-291ade00-4704-11eb-98cd-fa72c1234bea.png)

Green - incoming (You may pay)
Blue - paid
Red - Pending on declimed

You can pay/declime by press button (green) or click on invoice header and see more info about this (about invoice)
![image](https://user-images.githubusercontent.com/1880211/103141376-87e05780-4704-11eb-9c32-4da126be80bc.png)

P.S. Please also note that we use this system internally, and we do not fix some obvious bugs due to low priority (like not displaying the amount in usd or not displaying a link to a payment transaction) - we know about them and will try to fix / implement them as soon as possible opportunities, I hope for your understanding ;-)

**Comment by shendel** [comment]
before combat use, I highly recommend (highly) that you familiarize yourself with the function on testnet
For test it on testnet with testnet coin - select some latest pull in out repo (for example this pull https://github.com/swaponline/MultiCurrencyWallet/pull/3889)
In comments you can find answer from build bot, like this 
![image](https://user-images.githubusercontent.com/1880211/103141479-cf1b1800-4705-11eb-99cc-0a3e7a847b83.png)

select "testnet" link - and test there

**Comment by shendel** [comment]
@vptteam In a test environment, you can choose to recover from 12 words (there will automatically be words with test coins) or create your own test account

**Comment by vptteam** [comment]
great stuff.

**Comment by vptteam** [comment]
Working. Thanks for this update. @shendel
